Formula/p/pixi-pack.rb

```diff
@@ -0,0 +1,26 @@
+class PixiPack < Formula
+  desc "Pack and unpack conda environments created with pixi"
+  homepage "https://github.com/quantco/pixi-pack"
+  url "https://github.com/quantco/pixi-pack/archive/refs/tags/v0.3.1.tar.gz"
+  sha256 "d0c6eb4d20747a5ea47093dc06e80e15a46ec3fd9c3c1e93ee035480e3b0a75a"
+  license "BSD-3-Clause"
+  head "https://github.com/quantco/pixi-pack.git", branch: "main"
+
+  depends_on "cmake" => :build
+  depends_on "pkgconf" => :build
+  depends_on "rust" => :build
+
+  uses_from_macos "bzip2"
+
+  on_linux do
+    depends_on "openssl@3"
+  end
+
+  def install
+    system "cargo", "install", *std_cargo_args
+  end
+
+  test do
+    assert_equal "pixi-pack #{version}", shell_output("#{bin}/pixi-pack --version").strip
```

Comment:
We need a test that exercises the some of the functionality of the app. Version checks or usage checks (foo --version or foo --help) are not sufficient, as explained in the formula cookbook.

In most cases, a good test would involve running a simple test case: run #{bin}/foo input.txt.

- Then you can check that the output is as expected (with assert_equal or assert_match on the output of shell_output)
- You can also check that an output file was created, if that is expected: assert_predicate tepixi-packath/"output.txt", :exist?

Some advice for specific cases:

- If the formula is a library, compile and run some simple code that links against it. It could be taken from upstream's documentation / source examples.
- If the formula is for a GUI program, try to find some function that runs as command-line only, like a format conversion, reading or displaying a config file, etc.
- If the software cannot function without credentials, a test could be to try to connect with invalid credentials (or without credentials) and confirm that it fails as expected.
- Same if the software requires a virtual machine, docker instance, etc. to be running.